eng/SnippetGenerator/Program.cs

```diff
@@ -53,31 +52,37 @@ public async Task OnExecuteAsync()
             }
             else
             {
-                Console.WriteLine(message);
+                WriteError(message);
             }
             sw.Stop();
             Console.WriteLine($"SnippetGenerator completed in {sw.Elapsed}");
         }
 
         public static int Main(string[] args)
         {
-            ConsoleColor foreground = Console.ForegroundColor;
-
             try
             {
                 return CommandLineApplication.Execute<Program>(args);
             }
             catch (Exception e)
             {
-                Console.ForegroundColor = ConsoleColor.Red;
-
-                Console.Error.WriteLine(e.ToString());
+                WriteError(e.ToString());
```

Comment:
I realize this prints to stdout instead of stderr, but the error in the changes above also print to stdout. I don't see a clear reason why we should use stderr for a fatal error anyway. Not everyone knows how to redirect both stdout and stderr.